### DISCUSSION TITLE
Allowing 7z files on the site?

### DISCUSSION THREAD

**Discussion by Brume-AL** [atari-legend/atari-legend#203] [General]
Hi all,
I wanted to start to upload some RAW and SCP dumps on the site, but I see we can't upload 7z packed files. For large dumps, maybe could we use this format instead of ZIP format?
For instance, here is the weight of 'Gods disk A' in RAW format:
- ZIP --> 14,1 Mo (14 831 493 octets)
- 7z --> 10,0 Mo (10 583 503 octets)
Since we have more than 2500 games that are already dumped (around 5000 or 6000 disks I guess), it could lighten our web space if we use 7z for RAW and probably SCP files, which are huge, too.
Let me know.
Thanks in advance,
Brume

**Comment by nguillaumin** [comment]
There's 2 aspects:
- When a dump is uploaded in CPANEL, we always unzip it to check the content, find the `.st`, `.stx` or `.msa` file inside, and then re-zip it for storage and download
- For download, we provide the ZIP that was generated by CPANEL

So we need to consider both sides:
- Do we support 7z upload for CPANEL users? Ultimately it doesn't matter too much I think because uploading 14.1MB or 10MB is not  big change
- Do we store the files in 7z instead of Zip? In which case we should try to do it for everything to avoid having separate formats depending on the type

Personally for now I would say Zip is enough, until we run out of space and then we can tackle it?

**Reply by stefanjl** [reply]
Now i don't know if this have been decided already since the thread is a bit old... but i would prefer 7z also. 
AFAIK so does everyone that uses (and dump) flux copies use 7z so why would not AL support it?
